charts/jaeger/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.16.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.18.3
+version: 0.18.4
```

Comment:
```suggestion
version: 0.19.0
```

Reply:
For now, Let it be `0.18.4`, I have added **Ingester** support also to this chart. I will create another PR for that. We can change the version to `0.19.0` in that.

I'm currently trying out the Ingester part. I will make that PR next week hopefully :)